colour/models/rgb/transfer_functions/log.py

```diff
@@ -96,38 +110,52 @@ def log_decoding_Log2(log_norm,
                       min_exposure=0.18 * 2 ** -6.5,
                       max_exposure=0.18 * 2 ** 6.5):
     """
-    Defines the *Log2 shaper* log decoding function.
+    Defines the common *Log2* decoding function.
 
     Parameters
     ----------
     log_norm : numeric or array_like
                Logarithmic data to undergo decoding.
     middle_grey : numeric, optional
-                  'Middle Grey' exposure value.
+               *Middle Grey* exposure value.
     min_exposure : numeric, optional
-                   Minimum exposure level.
+               Minimum exposure level.
     max_exposure : numeric, optional
-                   Maximum exposure level.
+               Maximum exposure level.
 
     Returns
     -------
     numeric or ndarray
-        Linear *Log2 shaper* decoded data
+        Linear *Log2* decoded data.
 
     Notes
     -----
 
+    The common *Log2* decoding function can be used
+    to build logarithmic to linear shapers in the
+    ACES OCIO configuration.
+
+    The shaper with logarithmic encoded values can be
+    decoded back to linear domain:
+
+    +-------------------+------------------+
+    | **Shaper Range** | **Shaper Domain** |
```

Comment:
Careful, the table will break at the column separator are not aligned, sphinx is very picky with that.

Reply:
Will take care, Thankyou!